skimage/util/_map_array.py

```diff
@@ -32,7 +44,7 @@ def map_array(input_arr, input_vals, output_vals, out=None):
     # arr.reshape(-1) may be preferable."
     input_arr = input_arr.reshape(-1)
     if out is None:
-        out = np.empty(orig_shape, dtype=output_vals.dtype)
+        out = np.zeros(orig_shape, dtype=output_vals.dtype)
```

Comment:
```suggestion
        out = np.empty(orig_shape, dtype=output_vals.dtype)
```
See https://github.com/scikit-image/scikit-image/pull/7266#discussion_r1428006664.